ios/Plugin/Plugin.swift

```diff
@@ -137,7 +206,22 @@ public class RadarPlugin: CAPPlugin, RadarDelegate {
 
     @objc func getLocation(_ call: CAPPluginCall) {
         DispatchQueue.main.async {
-            Radar.getLocation { (status: RadarStatus, location: CLLocation?, stopped: Bool) in
+            let desiredAccuracy = call.getString("desiredAccuracy") ?? "medium";
+            var accuracy = RadarTrackingOptions.desiredAccuracy(for:"medium");
+            
+            switch desiredAccuracy.lowercased() {
+            case "high":
+                accuracy = RadarTrackingOptions.desiredAccuracy(for:"high")
+            case "medium":
+                accuracy = RadarTrackingOptions.desiredAccuracy(for:"medium")
+            case "low":
+                accuracy = RadarTrackingOptions.desiredAccuracy(for:"low")
+            default:
+                call.reject("bad request")
```

Comment:
Let's have the error message be:
`"invalid desiredAccuracy: " + desiredAccuracy`